.travis.yml

```diff
@@ -59,9 +60,9 @@ jobs:
         - CMD="$CMD1 && $CMD2"
         - USE_MAVEN_REPO="true"
 
-    - jdk: openjdk11
-      env:
+    - env:
         - DESC="NoErrorTest - Hibernate Search"
+        - CUSTOM_JDK_VERSION="temurin-17-jdk"
```

Comment:
Let's add adoptium to name as it is custom jdk from certain provider.

Reply:
done